cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -19,6 +20,77 @@
 from .services import GCESecurityService
 
 
+class GCPResourceUrl(object):
+
+    def __init__(self, resource, connection):
+        self._resource = resource
+        self._connection = connection
+        self.parameters = {}
+
+    def get(self):
+        discovery_object = getattr(self._connection, self._resource)()
+        return discovery_object.get(**self.parameters).execute()
+
+
+class GCPResources(object):
+    
+    def __init__(self, connection):
+        self._connection = connection
+
+        # Sorry, but the most reliable source of resource descriptions is the
```

Comment:
I don't think you need to apologize.
Can you provide a short example of what the content looks like (or is there a document/example online we could link to)?